build/commands/lib/config.js

```diff
@@ -650,10 +650,8 @@ Config.prototype.update = function (options) {
     this.is_asan = false
   }
 
-  if (options.use_goma && options.use_goma !== false) {
-    this.use_goma = true
-  } else {
-    this.use_goma = false
+  if (options.use_goma !== undefined) {
```

Comment:
this doesn't allow you to use use_goma=false to override npm config

Reply:
it does

Reply:
have you added `brave_use_goma=true` to npmrc? this doesn't work.